### PR TITLE
fix: various typos

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

File: files/en-us/web/css/@font-face/index.md [hunk 1]
```diff
@@ -90,9 +90,9 @@ The **`@font-face`** [CSS](/en-US/docs/Web/CSS) [at-rule](/en-US/docs/Web/CSS/At
     | `embedded-opentype` | Embedded OpenType     | .eot              |
     | `svg`               | SVG Font (deprecated) | .svg, .svgz       |
 
-    > **Note:** The values `opentype` and `truetype` are completely equivalent, regardless of whether the font file uses cubic bezier curves (within CFF/CFF2 table) or quadratic bezier curves (within glyf table).
+    > **Note:** The values `opentype` and `truetype` are completely equivalent, regardless of whether the font file uses cubic bezier curves (within CFF/CFF2 table) or quadratic bezier curves (within glyph table).
 
-    `tech()`: **Optional**. Not yet widely supported. Value is a _keyword_, support: `variations`, `palettes`, `incremental`, `features-opentype`, `features-aat`, `features-graphite`, `color-COLRv0`, `color-COLRv1`, `color-SVG`, `color-sbix`, `color-CBDT`.
+    `tech()`: **Optional**. Not yet widely supported. Value is a _keyword_, support: `variations`, `palettes`, `incremental`, `features-opentype`, `features-aat`, `features-graphite`, `color-colrv0`, `color-colrv1`, `color-svg`, `color-sbix`, `color-cbdt`.
```
Comment (teoli2003) [inline files/en-us/web/css/@font-face/index.md:95]: "Not yet widely supported" will not age well. Replaced by `{{Experimental_inline}}.

Also, the capitalization used is the one in the spec, putting it back.
```suggestion
    `tech()`: **Optional**. {{experimental_inline}} Value is one of the following _keyword_: `variations`, `palettes`, `incremental`, `features-opentype`, `features-aat`, `features-graphite`, `color-COLRv0`, `color-COLORv1`, `color-SVG`, `color-sbix`, `color-CBDT`.
```

**Review by teoli2003 — DISMISSED** [review]

File: files/en-us/web/css/@font-face/index.md (hunk 1 above)
Comment (nschonni) [reply files/en-us/web/css/@font-face/index.md:95]: The only other place that talks about these in the repo has them lowercase https://github.com/mdn/content/blob/main/files/en-us/web/css/@supports/index.md#font-tech

Comment (hamishwillee) [reply files/en-us/web/css/@font-face/index.md:95]: I don't _think_ the lower case version is a valid keyword. So I think the fix is to change the lists in https://github.com/mdn/content/blob/main/files/en-us/web/css/@supports/index.md#font-tech to match the spec.

Comment (nschonni) [reply files/en-us/web/css/@font-face/index.md]: I applied the `{{experimental_inline}}` part, but I left the casing part while the correct version is being confirmed

Comment (bsmth) [reply files/en-us/web/css/@font-face/index.md]: The specs mention [case insensitivity for keywords](https://www.w3.org/TR/css3-values/#keywords) but consistency would be nice. I would +1 with the suggestion from Hamish to have the list under `font-tech()` match.

File: files/en-us/web/css/@font-face/index.md [hunk 2]
```diff
@@ -90,9 +90,9 @@ The **`@font-face`** [CSS](/en-US/docs/Web/CSS) [at-rule](/en-US/docs/Web/CSS/At
     | `embedded-opentype` | Embedded OpenType     | .eot              |
     | `svg`               | SVG Font (deprecated) | .svg, .svgz       |
 
-    > **Note:** The values `opentype` and `truetype` are completely equivalent, regardless of whether the font file uses cubic bezier curves (within CFF/CFF2 table) or quadratic bezier curves (within glyf table).
+    > **Note:** The values `opentype` and `truetype` are completely equivalent, regardless of whether the font file uses cubic bezier curves (within CFF/CFF2 table) or quadratic bezier curves (within glyph table).
 
-    `tech()`: **Optional**. Not yet widely supported. Value is a _keyword_, support: `variations`, `palettes`, `incremental`, `features-opentype`, `features-aat`, `features-graphite`, `color-COLRv0`, `color-COLRv1`, `color-SVG`, `color-sbix`, `color-CBDT`.
+    `tech()`: **Optional**. {{Experimental_inline}} Value is a one of the following _keywords_: `variations`, `palettes`, `incremental`, `features-opentype`, `features-aat`, `features-graphite`, `color-colrv0`, `color-colrv1`, `color-svg`, `color-sbix`, `color-cbdt`.
```
Comment (bsmth) [inline files/en-us/web/css/@font-face/index.md:95]:
```suggestion
    `tech()`: **Optional**. {{Experimental_inline}} Value is a one of the following _keywords_: `variations`, `palettes`, `incremental`, `features-opentype`, `features-aat`, `features-graphite`, `color-COLRv0`, `color-COLORv1`, `color-SVG`, `color-sbix`, `color-CBDT`.
```

Comment (bsmth) [reply files/en-us/web/css/@font-face/index.md:95]: Adding back the previous suggestion

**Review by estelle — APPROVED** [review]

File: files/en-us/web/css/@font-face/index.md [hunk 3]
```diff
@@ -90,9 +90,9 @@ The **`@font-face`** [CSS](/en-US/docs/Web/CSS) [at-rule](/en-US/docs/Web/CSS/At
     | `embedded-opentype` | Embedded OpenType     | .eot              |
     | `svg`               | SVG Font (deprecated) | .svg, .svgz       |
 
-    > **Note:** The values `opentype` and `truetype` are completely equivalent, regardless of whether the font file uses cubic bezier curves (within CFF/CFF2 table) or quadratic bezier curves (within glyf table).
+    > **Note:** The values `opentype` and `truetype` are completely equivalent, regardless of whether the font file uses cubic bezier curves (within CFF/CFF2 table) or quadratic bezier curves (within glyph table).
 
-    `tech()`: **Optional**. Not yet widely supported. Value is a _keyword_, support: `variations`, `palettes`, `incremental`, `features-opentype`, `features-aat`, `features-graphite`, `color-COLRv0`, `color-COLRv1`, `color-SVG`, `color-sbix`, `color-CBDT`.
+    `tech()`: **Optional**. {{Experimental_inline}} Value is a one of the following _keywords_: `variations`, `palettes`, `incremental`, `features-opentype`, `features-aat`, `features-graphite`, `color-COLRv0`, `color-COLORv1`, `color-SVG`, `color-sbix`, `color-CBDT`.
```
Comment (bsmth) [inline files/en-us/web/css/@font-face/index.md:95]:
```suggestion
    `tech()`: **Optional**. {{Experimental_inline}} Value is a one of the following _keywords_: `variations`, `palettes`, `incremental`, `features-opentype`, `features-aat`, `features-graphite`, `color-COLRv0`, `color-COLRv1`, `color-SVG`, `color-sbix`, `color-CBDT`.
```

**Pull request merged**